apps/hashdotdev/src/pages/roadmap/technology-tree-data.tsx

```diff
@@ -0,0 +1,789 @@
+import { ReactNode } from "react";
+
+import { StatusId } from "./statuses";
+import { UseCaseId } from "./use-cases";
+import { VariantId } from "./variants";
+
+export type TechnologyTreeNodeData = {
+  id: string;
+  heading: ReactNode;
+  body?: ReactNode;
```

Comment:
I can remove this yes, the reason why I added it was to allow @nonparibus to use bold/italics in the copy. If we don't for-see needing this I can make this a `.ts` file and constrain the type.

Reply:
It might be useful. Suggest we leave to hedge.

Reply:
Let's update (or not) after we've agreed on the copy. It's better not to have files lying around called `tsx` that don't have any components in them because it makes it harder to understand the contents of the folder